src/components/wizard/wizard-step.component.ts

```diff
@@ -64,18 +69,33 @@ export class WizardStepComponent {
         this._changeDetector.markForCheck();
     }
 
-    @HostBinding('attr.aria-expanded')
+    /** Emits when visited changes. */
+    @Output() visitedChange = new EventEmitter<boolean>();
+
     get active(): boolean {
         return this._active;
     }
 
     constructor(
-        private readonly _changeDetector: ChangeDetectorRef,
-        private readonly _elementRef: ElementRef,
-        private readonly _renderer: Renderer2) { }
+        private readonly _wizardService: WizardService<WizardStepComponent>,
+        private readonly _changeDetector: ChangeDetectorRef) {
+    }
+
+    setVisited(value: boolean): void {
```

Comment:
To make this super clear and prevent future misunderstandings which mess up our outputs, I'd like this and any future methods which do the same thing to be named like:

setVisitedAndEmitChangeEvent

That way someone won't accidentally use this in a setter for `visited`, for example.